examples/seismic/tti/operators.py

```diff
@@ -390,7 +390,7 @@ def ForwardOperator(model, source, receiver, space_order=4,
     ang1 = sin(theta)
     ang2 = 0
     ang3 = 0
-    if len(model.shape) == 3:
+    if len(model.shape) == 3 and phi is not None:
```

Comment:
should we rather raise an error otherwise? as in, if a 1D/2D problem and ``phi != None``...

Reply:
No 1D for TTI, tilted doesn't mean anything in 1D, can be done that way too will look at it

Reply:
Warning thrown in model, but this is still required to avoid `cos(None)` for a 3D without `phi`